### DISCUSSION TITLE
Resumable download of files

### DISCUSSION THREAD

**Discussion by whmeitzler** [pocketbase/pocketbase#5334] [Q&A] [answered]
Hello!

I'm going to need to download files from a server over a weak cellular connection, and would like to use pocketbase as that server.

In the case where a connection is interrupted, I'd like to resume the download from where I left off.

I understand that the standard way of doing this is [HTTP range requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests)

Does pocketbase support these requests when serving files from a collection?

Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, range file download requests are supported but note that the SDKs don't have anything builtin for it and you'll have to manage the "resumability" on your own.

**Reply by whmeitzler** [reply]
Wonderful! Managing my own download is totally fair.

Are there any other useful headers that Pocketbase supports? I'm most interested in resuming downloads, and accepting gzip (and possibly the interplay between the two)

Thanks again

**Reply by ganigeorgiev** [reply]
You could check the godoc of [`http.ServeContent`](https://pkg.go.dev/net/http#ServeContent) (_that is internally what is used for the files download_).
